processing/src/main/java/org/apache/druid/segment/vector/BitmapVectorOffset.java

```diff
@@ -85,6 +87,14 @@ public void advance()
 
       currentVectorSize = to;
     }
+
+    if (currentVectorSize > 1) {
```

Comment:
Why not do this branch if `currentVectorSize == 1`?

Reply:
we could, and I considered it, but ultimately I decided that 1 didn't really "feel" like it met the threshold for me to consider it really contiguous. I also figured it didn't really make a lot of difference between which read method it would end up in because there is only 1 offset either way. But I don't have strong opinions on it though since either seems valid, do you think it logically should be considered contiguous?

Reply:
I guess it doesn't really matter. I thought I might have been missing some reason why it wasn't ok to do it for the `currentVectorSize == 1` case.